### ISSUE TITLE
Suggestion/Request: Remote Server Connection

### ISSUE THREAD

**Issue by sahaq** [Samachiy/Qinti#38]
Hello!  I am currently testing Qinti and loving it, however I see an opportunity that most developers don't seem to want to integrate, or maybe it's just a lack of demand.  

I have a handful of friends and family who have access to my auto1111/comfy/sd.next instances and they are used pretty heavily, however even using the gradio auth methods available, UserA can always see UserBs generations, and with some front ends can even have there own images 'overwritten' in the preview pane when someone else does a generation.  Since this looks like it handles everything through the API, Would it be possible to interact with a remote server? 

Open-WebUI and SillyTavern both have options for image generation where everything is handled via an api connection, so I believe it should be possible, but I'm not going to pretend I understand what level of complexity is required.  

In summary; I'm requesting that an option be added on setup to instead of connect to local, or install local, include the option to connect to a remote instance, providing ip/port entries, so the application can be ran on PC1, but rendering is done on ServerA.  

Thanks for this application, and have a great day!

**Comment by Samachiy** [comment]
Hello. Glad to hear that you are liking the program so far! :)

Yes, it is possible, the only problem really, is that in those web UI APIs there is no endpoint to retrieve things like LoRA models, leaving no access to al the stuff in the "Resources tab". Because of this I think there are two options, to either ask and wait for the developer of the respective web UI to implement it for their side or from this side create a sort of Qinti compatibility API layer that will handle that stuff. The latter sound more reliable to implement, but I don't know what would be the most comfortable way for the owner of the instance to set it up for remote use. 

If I'm not mistaken, most of the time the command line is preferable for setting up something in the server rather than an interface? If not, what do you think would be the option that fits better and be more comfortable to set up server side?
